### DISCUSSION TITLE
Documentation

### DISCUSSION THREAD

**Discussion by MohamedElashri** [wherkamp/nitro_repo#182] [Ideas]
Hi this projects needs some docs on what are the parameters and scope. How to use it, self-host and build binaries ...etc. 

Also, some screenshots on the README.md page would be very useful

**Comment by wyatt-herkamp** [comment]
I hope to have a public dev builds and information on setting it up this week. Thank you for the interest in the project.

**Comment by wyatt-herkamp** [comment]
@MohamedElashri I know this is not the greatest. But hopefully it is a start. https://kingtux.dev/nitro_repo/getting-started/
